.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules/
 bower_components/
 config/config.js
 rise-financial-version.js
+rise-financial.html
```

Comment:
@settinghead That sounds like a great idea! @stulees I think we should do that instead of committing those files to Github.